gap/almostsimple.gi

```diff
@@ -21,10 +21,10 @@
 RECOG.ParseNumber := function( number, d, default )
   if IsInt(number) then
       return number;
-  fi;
-  if IsString(number) then
-      if number = "logd" then return LogInt(d,2); fi;
-      if number[Length(number)] = 'd' then
+  elif IsString(number) then
+      if number = "logd" then
+          return LogInt(d, 2);
+      elif number[Length(number)] = 'd' then  # Can eventually be Last(number)
```

Comment:
You could further compact this to:  `if IsInt(number) ... elif number = "logd" ... elif IsString(number) and EndsWith(number, "d") ... fi;` (this also avoids the use of `Last`)

Reply:
Ah, nice suggestion.